requirements.txt

```diff
@@ -13,6 +13,7 @@ pytz==2020.1
 zstandard
 authlib
 apscheduler
+werkzeug
```

Comment:
werkzeug is a dependency of Flask already: https://github.com/pallets/flask/blob/aa1d34dc51fc63dabc46f388822ba73b15d3ef97/setup.py#L7

So you can remove this line again.

Reply:
Or maybe it's good to make it explicit? I don't know

Reply:
I think we can keep it explicit, just cause we're actually importing it.

Reply:
After personal discussion, we decided to keep for now, but are not strongly opinionated on the issue.